elfi/methods/parameter_inference.py

```diff
@@ -941,54 +940,115 @@ def _report_batch(self, batch_index, params, distances):
             str += "{}{} at {}\n".format(fill, distances[i].item(), params[i])
         logger.debug(str)
 
-    def plot_state(self, **options):
-        """Plot the GP surface
-        
-        Currently supports only 2D cases.
+    def plot_acq_points(self):
+        """Plot the acquisition points.
```

Comment:
You could add here that this is still experimental because all of these `plot_state` methods still are. It would be a nice feature if one could specify which 2 dimensions are plotted in higher dimensions and fix the other dimensions to certain values.